isis/version

```diff
@@ -1,3 +1,3 @@
 3.7.0        # Public version number
-2019-04-16   # Release date
-alpha        # release stage (alpha, beta, stable)
+2019-04-20   # Release date
```

Comment:
This should probably change to 4/30 or 5/1

Reply:
That was supposed to be 30! I will fix it now. Does 5/1 make more sense though (even if I get it out today)? Or should I wait, technically?

Reply:
I think 4/30 is just fine